src/App.php

```diff
@@ -265,13 +265,21 @@ protected function configureRoutes( Slim $slim ) {
 				$middleware['inject-user'], $middleware['set-environment'],
 			function () use ( $slim, $middleware, $routeConditions ) {
 				$slim->get( '', function() use ( $slim ) {
+					// See if we have a cookie indicating last version used
+					if ( isset( $_COOKIE['copypatrolLang'] ) ) {
```

Comment:
It doesn't make much difference here, but there's `$slim->getCookie()` that will do this as well. It returns null if there's no cookie.